native/src/components/SettingItem.tsx

```diff
@@ -48,25 +33,35 @@ const Description = styled.Text`
   color: ${props => props.theme.colors.textSecondaryColor};
 `
 
+type PropType = {
+  title: string
+  description?: string
+  onPress: () => void
+  t: TFunction<'settings'>
```

Comment:
When du you sill pass `t` and when du you use `useTranslations` in the component directly?

Reply:
I think I don't use `useTranslations` directly, do I?

Reply:
yes you do in a lot of components, therefore it is a litte inconsistent. Thats why i'm asking.

Reply:
ah okay, so more in general. In this specific component its not used though, so I have to pass it (or use the hook oc). Yeah we have no rule for that but it was also inconsistent before. I think this is not a problem though. My rule of thumb if I write code myself is to pass it if it is used in the parent component and use the hook if not used in the parent component or there are usages in different routes with different translation namespaces.